docs/get-started/_index.md

```diff
@@ -1,36 +1,146 @@
+my_motor = Motor.from_robot(robot=robot, name="my_motor")
+# Set the power to 40% forwards.
+await my_motor.set_power(power=0.4)
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go {class="line-numbers linkable-line-numbers"}
+myMotorComponent, err := motor.FromRobot(machine, "my_motor")
+// Set the motor power to 40% forwards.
+myMotorComponent.SetPower(context.Background(), 0.4, nil)
+```
+
+{{% /tab %}}
+{{% tab name="Flutter" %}}
+
+```dart {class="line-numbers linkable-line-numbers"}
+final base = Motor.fromRobot(client, "my_motor");
+// Set the power to  40% forwards.
+await myMotor.setPower(0.4);
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+```ts {class="line-numbers linkable-line-numbers"}
+const myMotor = new VIAM.MotorClient(client, "my_motor");
+// Set the power to  40% forwards.
+await myMotor.setPower(0.4);
+```
 
-#### Community
+{{% /tab %}}
+{{% tab name="C++" %}}
 
-Have questions, or want to meet other people working on smart machines? Join us in the Community Discord!
+```cpp {class="line-numbers linkable-line-numbers"}
+std::shared_ptr<Motor> motor = robot->resource_by_name<Motor>("my_motor");
+// Set the power to  40% forwards.
+motor->set_power(0.4);
+```
 
-{{% /manualcard %}}
+{{% /tab %}}
+{{< /tabs >}}
 
+The standardized nature of Viam's resource APIs means that when you build machines, you can swap out components such as motors without changing any code.
+The only change needed is to your machine configuration.
+The Viam Registry provides a marketplace for adding and sharing modules, all using these standard APIs.
+
+However, if you need more custom behaviour, you can extend these APIs to suit your own needs.
+
+You can use Viam's SDKs, as well as the Viam app, and the Viam mobile app, to control and monitor your machines securely.
+If `viam-server` can connect to the internet, you can control and monitor your machines securely from anywhere in the world.
+In scenarios where machines are connected to a local area network (LAN) with intermittent or no cloud connectivity, you can control and monitor machines from the local network or with code on the machines themselves.
+
+With the features so far, you can configure a wide variety of machines.
+The Viam app does not just provide a useful configuration interface.
+It also provides several higher-level functionalities to make your machines smarter and better, such as:
+
+- **Data Management**: Any data on your machine can be synced to the cloud, from where you can query it using SQL, MQL, or with code.
+- **Machine Learning**: Train machine learning models on collected data and deploy ML models to machines to enable them to intelligently see and interpret the world around them.
+- **Simultaneous Localization And Mapping (SLAM)**: A machine can map its surroundings and find its position on a map.
+
+On a more structural level, the [Viam App](https://app.viam.com)'s **fleet management** capabilities enable you to:
```

Comment:
I think, to improve scanning and reduce verbosity, this could be
```suggestion
Viam app fleet management capabilities:
```
Then users don't have to wonder what we mean by "a more structural level" and can just get into it.

```suggestion
On a more structural level, the [Viam app](https://app.viam.com)'s **fleet management** capabilities enable you to:
```
[nit] if this stays